debian/control

```diff
@@ -2,7 +2,7 @@ Source: py-libzfs
 Section: contrib/python
 Priority: optional
 Maintainer: William Grzybowski <william@grzy.org>
-Build-Depends: debhelper-compat (= 12),
+Build-Depends: debhelper,
```

Comment:
Is this change really needed? I working as a debian maintainer and all developers tell me using debhelper-compat is the correct way to do this

Reply:
I've changed it since apt says that package does not exist. Only [debhelper](https://packages.debian.org/buster/debhelper).

Reply:
I think debhelper-compat is handled especially by dpkg-buildpackage, its not an actual package

Reply:
https://github.com/luqasz/py-libzfs/commit/86acf15a924c5af9a206fbeb88cb063f05df45b6/checks?check_suite_id=404470318

`dpkg-buildpackage` complains about `debhelper-compat`